libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -245,6 +245,7 @@ const ParentalLeaveTemplate: ApplicationTemplate<
                   Promise.resolve(val.EmployerApproval),
                 ),
               read: { answers: ['periods'], externalData: ['pregnancyStatus'] },
+              write: 'all',
```

Comment:
We should not give `ASSIGNEE` a `write: 'all'` permission.

Reply:
True, was using it as hotfix 🙃. Will investigate further.

Reply:
I investigated it further, and it was related to the new structure for `selectedChild` and `children` externalData. 
The latest commit should fix it more cleanly. I'll take a look if we can have a test for this, otherwise should be ready for re-review.